squid-blockexplorer/src/blocks/utils.ts

```diff
@@ -99,7 +99,7 @@ export function createCall(
 export function calcSpacePledged(solutionRange: bigint): bigint {
   const MAX_U64 = 2n ** 64n - 1n;
   const SLOT_PROBABILITY = [1n, 6n];
-  const PIECE_SIZE = 31744n;
+  const PIECE_SIZE = 65536n;
```

Comment:
Shouldn't `PIECE_SIZE` be the same in `calcSpacePledged` and `calcHistorySize`? Is there a reason not to set these outside of the functions and share the constants?

Reply:
Is the same, one is just being calculated, will update shortly